### DISCUSSION TITLE
[ SOLVED ] How to use a fill in contract

### DISCUSSION THREAD

**Discussion by unefined** [wevm/wagmi#2524] [Q&A]
Hi all, someone can tell me how i can fill on contract this fill or multiple fills ?
Please help, thanks!

Code contract usage
```js
const { config } = usePrepareContractWrite({
        address: '0xf195855d48aCFe13e9EAAdDD80c3611bC7487657',
        abi: [
        {
            name: 'mint',
            type: 'function',
            stateMutability: 'nonpayable',
            inputs: [],
            outputs: [],
        },
        ],
        functionName: 'mint',
    })


    const { write: mint } = useContractWrite(config);
```

i am using rainbowkit if this is important

upd: sorry i forgot give a screenshot
https://imgur.com/a/DAuBfhK

https://goerli.etherscan.io/address/0xf195855d48acfe13e9eaaddd80c3611bc7487657#writeContract

**Comment by ombayoeco11** [comment]
const { config } = usePrepareContractWrite({
        address: '0xf195855d48aCFe13e9EAAdDD80c3611bC7487657',
        abi: [
        {
            name: 'mint',
            type: 'function',
            stateMutability: 'nonpayable',
            inputs: [],
            outputs: [],
        },
        ],
        functionName: 'mint',
       args:[] //input any number  
    })


    const { write: mint } = useContractWrite(config);

**Reply by unefined** [reply]
type "number[]" cannot be set as "undefined"
something like this.

**Reply by unefined** [reply]
try, this don't work

**Comment by unefined** [comment]
sorry i forgot give a screenshot
https://imgur.com/a/DAuBfhK

**Comment by unefined** [comment]
SOLVED: Checked documentation wagmi examples

**Reply by ombayoeco11** [reply]
sorry if i'm not helping...

**Reply by unefined** [reply]
u was right with args[], but was needing input json object
